code/game/verbs/suicide.dm

```diff
@@ -229,3 +229,19 @@
 		setCloneLoss(100, FALSE)
 
 		updatehealth()
+
+/mob/living/simple_animal/mouse/verb/suicide()
+	set hidden = 1
+	if(stat == 2)
+		to_chat(src, "You're already dead!")
+		return
+	if(suiciding)
+		to_chat(src, "You're already committing suicide! Be patient!")
+		return
+
+	var/confirm = alert("Are you sure you want to commit suicide?", "Confirm Suicide", "Yes", "No")
+
+	if(confirm == "Yes")
+		suiciding = 1
+		to_chat(viewers(src), "<span class='danger'>[src] is playing dead permanently! It looks like [p_theyre()] trying to commit suicide.</span>")

```

Comment:
Isn't `visible_message` used for sending a chat to viewers?

Reply:
Yeeee

Reply:
I'm not actually sure, since I was just using existing code in the suicide.dm for reference- is there a way I could use visible_message to make it work better?

Reply:
I think it'd literally just be...
`visible_message("<span class='danger'>[src] is playing dead permanently! It looks like [p_theyre()] trying to commit suicide.</span>")`

Reply:
give the comments here a read: https://github.com/ParadiseSS13/Paradise/blob/10ef2311ab0b5283422da0ba63783ab97430e1e2/code/modules/mob/mob.dm#L99

Reply:
It worked perfect, so I went ahead and fixed it!